### ISSUE TITLE
Exporting SRT to a database friendly format

### ISSUE THREAD

**Issue by Anan5a** [captioning/captioning#76]
This is a feature request
I want to implement this library into a translation tool. which will help translating subtitle by a group collaboration. So I wanted to export base subtitle to a format which can be easily stored on database (mysql)
eg. ``` I want to store the cue number, timecode, cue text into separate column ```

**Comment by fernando-ayon** [comment]
@Anan5a I don't think this should be part of this library as it is none of its concerns.
This library allows you to parse and create subtitles files, but does not care about databases. Using the methods available from this library, you can create your own implementation of that.
